content/inside-rust/stage0-redesign/index.md

```diff
@@ -0,0 +1,70 @@
++++
+path = "inside-rust/9999/12/31/redesigning-stage-0-std"
+title = "Redesigning stage 0 std"
+authors = ["Jieyou Xu"]
+
+[extra]
+team = "Bootstrap"
+team_url = "https://www.rust-lang.org/governance/teams/infra#team-bootstrap"
++++
+
+# Redesigning stage 0 std
+
+## Summary
+
+We are reworking how the stage 0 bootstrap sequence works (the sequence used to build a stage 1 compiler).
+
+- Before: a stage 0 (beta by default) compiler is used to build the in-tree std, which in turn is used to build the stage 1 compiler.
+- After: a stage 0 (beta by default) compiler and a precompiled stage 0 std is instead used to build the stage 1 compiler.
+
```

Comment:
I wonder if this could do with a short summary with less of the... I hesitate to say jargon, but something like:
```suggestion
- Before: a stage 0 (beta by default) compiler is used to build the in-tree std, which in turn is used to build the stage 1 compiler.
- After: a stage 0 (beta by default) compiler and a precompiled stage 0 std is instead used to build the stage 1 compiler.

In other words, in the typical workflow, the stage1 compiler now depends on beta std instead of the in-tree std (built by the beta compiler).
```
Generally I find the current "before/after" description to be a little bit hard to parse :thinking: